.github/workflows/continuous_integration.yml

```diff
@@ -14,8 +12,14 @@ jobs:
     - name: Build container
       run: make build
     
-    - name: Start container 
-      run: source .env; make run
+    - name: Copy environment 
+      run: cp -v .env database.env .github/startup-test
```

Comment:
If you follow the suggestion from above and move the dc.yml file back, then you shouldn't need this command anymore. But instead you'll have to create the secret files, see other comment above